### PR TITLE
Update new character creation

### PR DESCRIPTION
**Pull request by jazeee**
Only show new character creation if there is:
1. No race
2. No skills

Also, only show `Add Base Skills` if there are no skills.

Adjust styles in UI.

Change `Custom` to `Other`
Make sure other does not add base skills.

Remove image avatar, and adjust styling to ensure aspect ratio.

https://github.com/epic-power-rpg/roll20/assets/5629800/3ab17913-9d93-4d9e-bdf0-be51be944e2a

### CODE REVIEW

File: bin/main.css [hunk 1]
```diff
@@ -109,7 +109,8 @@ div.popoverContainer > .fixedPlacementPopover {
   gap: 16px;
```
Comment (jazeee) [inline bin/main.css:109]: This is stacked on #102 
That is a simple URL fix.

**Review by johnlamping — COMMENTED** [review]
I think you need  to add
on('sheet:opened',  updateUserStatistics)

Otherwise isNewUser won't be right for old characters.

**Comment by jazeee** [comment]
It is being triggered through the on update for skills which then updates
this.
It is a bit of a chain of command, but the callback flow is wired up.

**Review by johnlamping — APPROVED** [review]
Got it. Thanks.

**Pull request merged**